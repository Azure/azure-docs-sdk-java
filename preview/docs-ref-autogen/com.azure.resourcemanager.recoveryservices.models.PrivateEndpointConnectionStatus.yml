### YamlMime:JavaType
uid: "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus"
fullName: "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus"
name: "PrivateEndpointConnectionStatus"
nameWithType: "PrivateEndpointConnectionStatus"
summary: "Defines values for PrivateEndpointConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateEndpointConnectionStatus extends ExpandableStringEnum<PrivateEndpointConnectionStatus>"
constructors:
- "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus.PrivateEndpointConnectionStatus()"
fields:
- "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus.APPROVED"
- "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus.DISCONNECTED"
- "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus.PENDING"
- "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus.REJECTED"
methods:
- "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.recoveryservices.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-recoveryservices:1.0.0-beta.1
