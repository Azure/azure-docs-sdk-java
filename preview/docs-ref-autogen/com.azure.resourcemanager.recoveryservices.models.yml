### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.recoveryservices.models"
fullName: "com.azure.resourcemanager.recoveryservices.models"
name: "com.azure.resourcemanager.recoveryservices.models"
summary: "Package containing the data models for RecoveryServicesManagementClient. Recovery Services Client."
classes:
- "com.azure.resourcemanager.recoveryservices.models.AuthType"
- "com.azure.resourcemanager.recoveryservices.models.CertificateRequest"
- "com.azure.resourcemanager.recoveryservices.models.CheckNameAvailabilityParameters"
- "com.azure.resourcemanager.recoveryservices.models.ClientDiscoveryDisplay"
- "com.azure.resourcemanager.recoveryservices.models.ClientDiscoveryForLogSpecification"
- "com.azure.resourcemanager.recoveryservices.models.ClientDiscoveryForProperties"
- "com.azure.resourcemanager.recoveryservices.models.ClientDiscoveryForServiceSpecification"
- "com.azure.resourcemanager.recoveryservices.models.ClientDiscoveryResponse"
- "com.azure.resourcemanager.recoveryservices.models.IdentityData"
- "com.azure.resourcemanager.recoveryservices.models.JobsSummary"
- "com.azure.resourcemanager.recoveryservices.models.MonitoringSummary"
- "com.azure.resourcemanager.recoveryservices.models.NameInfo"
- "com.azure.resourcemanager.recoveryservices.models.PatchTrackedResource"
- "com.azure.resourcemanager.recoveryservices.models.PatchVault"
- "com.azure.resourcemanager.recoveryservices.models.PrivateEndpoint"
- "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus"
- "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionVaultProperties"
- "com.azure.resourcemanager.recoveryservices.models.PrivateLinkResources"
- "com.azure.resourcemanager.recoveryservices.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.recoveryservices.models.ProvisioningState"
- "com.azure.resourcemanager.recoveryservices.models.RawCertificateData"
- "com.azure.resourcemanager.recoveryservices.models.ReplicationUsageList"
- "com.azure.resourcemanager.recoveryservices.models.ResourceCertificateAndAadDetails"
- "com.azure.resourcemanager.recoveryservices.models.ResourceCertificateAndAcsDetails"
- "com.azure.resourcemanager.recoveryservices.models.ResourceCertificateDetails"
- "com.azure.resourcemanager.recoveryservices.models.ResourceIdentityType"
- "com.azure.resourcemanager.recoveryservices.models.Sku"
- "com.azure.resourcemanager.recoveryservices.models.SkuName"
- "com.azure.resourcemanager.recoveryservices.models.TriggerType"
- "com.azure.resourcemanager.recoveryservices.models.UpgradeDetails"
- "com.azure.resourcemanager.recoveryservices.models.UsagesUnit"
- "com.azure.resourcemanager.recoveryservices.models.VaultList"
- "com.azure.resourcemanager.recoveryservices.models.VaultPrivateEndpointState"
- "com.azure.resourcemanager.recoveryservices.models.VaultProperties"
- "com.azure.resourcemanager.recoveryservices.models.VaultUpgradeState"
- "com.azure.resourcemanager.recoveryservices.models.VaultUsageList"
interfaces:
- "com.azure.resourcemanager.recoveryservices.models.CheckNameAvailabilityResult"
- "com.azure.resourcemanager.recoveryservices.models.ClientDiscoveryValueForSingleApi"
- "com.azure.resourcemanager.recoveryservices.models.Operations"
- "com.azure.resourcemanager.recoveryservices.models.PrivateLinkResource"
- "com.azure.resourcemanager.recoveryservices.models.PrivateLinkResourcesOperations"
- "com.azure.resourcemanager.recoveryservices.models.RecoveryServices"
- "com.azure.resourcemanager.recoveryservices.models.RegisteredIdentities"
- "com.azure.resourcemanager.recoveryservices.models.ReplicationUsage"
- "com.azure.resourcemanager.recoveryservices.models.ReplicationUsages"
- "com.azure.resourcemanager.recoveryservices.models.Usages"
- "com.azure.resourcemanager.recoveryservices.models.Vault"
- "com.azure.resourcemanager.recoveryservices.models.Vault.Definition"
- "com.azure.resourcemanager.recoveryservices.models.Vault.DefinitionStages"
- "com.azure.resourcemanager.recoveryservices.models.Vault.DefinitionStages.Blank"
- "com.azure.resourcemanager.recoveryservices.models.Vault.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.recoveryservices.models.Vault.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.recoveryservices.models.Vault.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.recoveryservices.models.Vault.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.recoveryservices.models.Vault.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.recoveryservices.models.Vault.DefinitionStages.WithSku"
- "com.azure.resourcemanager.recoveryservices.models.Vault.DefinitionStages.WithTags"
- "com.azure.resourcemanager.recoveryservices.models.Vault.Update"
- "com.azure.resourcemanager.recoveryservices.models.Vault.UpdateStages"
- "com.azure.resourcemanager.recoveryservices.models.Vault.UpdateStages.WithEtag"
- "com.azure.resourcemanager.recoveryservices.models.Vault.UpdateStages.WithIdentity"
- "com.azure.resourcemanager.recoveryservices.models.Vault.UpdateStages.WithProperties"
- "com.azure.resourcemanager.recoveryservices.models.Vault.UpdateStages.WithSku"
- "com.azure.resourcemanager.recoveryservices.models.Vault.UpdateStages.WithTags"
- "com.azure.resourcemanager.recoveryservices.models.VaultCertificateResponse"
- "com.azure.resourcemanager.recoveryservices.models.VaultCertificateResponse.Definition"
- "com.azure.resourcemanager.recoveryservices.models.VaultCertificateResponse.DefinitionStages"
- "com.azure.resourcemanager.recoveryservices.models.VaultCertificateResponse.DefinitionStages.Blank"
- "com.azure.resourcemanager.recoveryservices.models.VaultCertificateResponse.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.recoveryservices.models.VaultCertificateResponse.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.recoveryservices.models.VaultCertificateResponse.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.recoveryservices.models.VaultCertificates"
- "com.azure.resourcemanager.recoveryservices.models.VaultExtendedInfoResource"
- "com.azure.resourcemanager.recoveryservices.models.VaultExtendedInfoes"
- "com.azure.resourcemanager.recoveryservices.models.VaultUsage"
- "com.azure.resourcemanager.recoveryservices.models.Vaults"
metadata: {}
package: "com.azure.resourcemanager.recoveryservices.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-recoveryservices:1.0.0-beta.1
