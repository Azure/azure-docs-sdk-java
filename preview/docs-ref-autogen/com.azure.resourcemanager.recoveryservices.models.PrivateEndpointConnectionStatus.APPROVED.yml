### YamlMime:JavaMember
uid: "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.recoveryservices.models.PrivateEndpointConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointConnectionStatus."
  syntax: "public static final PrivateEndpointConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.recoveryservices.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-recoveryservices:1.0.0-beta.1
